adhocteam/Head-Start-TTADP
# Handle HSES resets

## Description of change
A change to the `findOrCreateUser `function to allow finding users by `hsesUsername` as an alternative if not found by `hsesUserId`. This change allows gracefully handling HSES resets in non-prod environments.

## How to test
Make sure tests pass 
or verify login works here
https://tta-smarthub-sandbox.app.cloud.gov/

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-119


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
